xbmc/addons/InputStream.cpp

```diff
@@ -98,7 +98,7 @@ bool CInputStream::Supports(CFileItem &fileitem)
   CVariant addon = fileitem.GetProperty("inputstreamaddon");
```

Comment:
wasn't that you who volunteered for the documentation job? :)
the idea for this request is less than 2 days old. still experimenting.